airmozilla/manage/views.py

```diff
@@ -3581,6 +3581,7 @@ def picturegallery_data(request):
     context['urls'] = {
         'manage:picture_edit': reverse('manage:picture_edit', args=('0',)),
         'manage:picture_delete': reverse('manage:picture_delete', args=('0',)),
+        'manage:picture_delete_all': reverse('manage:picture_delete_all'),
```

Comment:
With my comment earlier this'll need to become `reverse('manage:picture_delete_all', args=('0',))`